notifyme/candy.js

```diff
@@ -77,7 +79,11 @@ CandyShop.NotifyMe = (function(self, Candy, $) {
 			if (searchRegExp.test(args.templateData.message) && args.templateData.name != _getNick()) {
 				// highlight if specified
 				if (_options.highlightInRoom) {
-					args.templateData.message = args.templateData.message.replace(searchRegExp, '$1<span class="candy-notifyme-highlight">' + searchTerm + '</span>');
+					if (_options.normalizeNickname) {
+						args.templateData.message = args.templateData.message.replace(searchRegExp, '$1<span class="candy-notifyme-highlight">' + searchTerm + '</span>');
```

Comment:
It would be nice to remove the duplication here and leave only the content of the tag inside the conditional.